### ISSUE TITLE
Send message to teams

### ISSUE THREAD

**Issue by mikaelsvensson** [tunnelbanejakten/wp-tuja#8]
Add admin GUI support for sending text message, either by SMS or mail, to selected teams. Ability for teams to reply to message is not required.

Messages may contain these variables:
- NAME (of group)
- AGE_GROUP
- CONTACT_PHONE
- GROUP_PASSWORD (used to identify the group instead of username and password)
- MINUTES_LEFT (until the competition closes and no more responses are accepted)

Previewing messages before sending them (including character counters) is a necessity.

Stretch goal: Allow message templates to be created ahead of time.

**Comment by mikaelsvensson** [comment]
Start after #14 has been merged so that recipient teams can be selected by category.
